active/social-media.md

```diff
@@ -0,0 +1,69 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships/campaigns.
+
+The group opts to manage platforms that best achieve these goals - which may be a subset of where Django is or could be engaging.
+
+Delegated responsibilities:
+- The Chair can grant access to Django's official social media profiles and each member will have access.
+- Members can post content to Django's official social media profiles, in alignment with the working group's agreed goals, without a board review.
+- Members can like/repost/comment as Django, when it aligns with the working group's agreed goals, without a board review.
+- The working group can post content on djangoproject.com.
+
+Actions to take back to the Board for votes:
+- Amendments to the working group's goals.
+- Approval on content that is outside the agreed group's goals.
+- Proposals of social media campaigns for our sponsors.
+- Proposal to have a new social media profile on a platform.
+- Proposal to retire a social media profile from a platform.
+
+## Initial membership
+
+- Chair:
+- Co-Chair:
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Char):
+- Other members:
+
+## Future membership
+
+**Who is eligible to join? Any volunteer, or are there specific requirements?**
+
+Members must have some experience using at least one of Django's social media platforms (such as X).
```

Comment:
I’d keep it simpler – and make it easier to check new members by requiring Individual Membership. If someone isn’t an individual member but we otherwise think they would be a good fit, we can suggest specific contributions for them to demonstrate their commitment to contributing to Django and apply for membership once the time is right.

```suggestion
Members must have demonstrable experience with social media platforms, and be registered as a Django Software Foundation Individual Member.
```

Reply:
This is a good idea. We can send an email out in the DSF mailing list to gage interest in joining the group.